apps/coordinator/src/components/ScriptExplorer/SignatureImporter.jsx

```diff
@@ -482,13 +492,17 @@ class SignatureImporter extends React.Component {
               return;
             }
             try {
-              // This returns false if it completes with no error
-              publicKey = validateMultisigSignature(
+              const args = {
                 network,
-                inputs,
-                outputs,
+                inputs: inputs.map(convertLegacyInput),
+                outputs: outputs.map(convertLegacyOutput),
+              };
+              const psbt = getUnsignedMultisigPsbtV0(args);
+              publicKey = validateMultisigPsbtSignature(
+                psbt.toBase64(),
```

Comment:
this was a big change in the API. The main functions now operating past the creator role should take a psbt and can extract inputs/output data from there.